dex/msgjson/types.go

```diff
@@ -1176,6 +1176,8 @@ type EpochReportNote struct {
 	Epoch        uint64 `json:"epoch"`
 	BaseFeeRate  uint64 `json:"baseFeeRate"`
 	QuoteFeeRate uint64 `json:"quoteFeeRate"`
+	// quantity of matches by rate: [rate]qty
+	MatchesSummary map[uint64]uint64 `json:"matchessumary"`
```

Comment:
How about separating into buys and sells so we can apply colors eventually.

Reply:
The is one issue with aggregating.  Some are takers buying some are takers selling, combined in quantity at a given rate.

Reply:
You can keep separate maps/slices all the way down.

Reply:
Works for me.  To be clear you mean in `processEpochReady`, two maps server-side, one for aggregated taker sells and one for aggregated taker buys?
`match.Taker.Trade().Sell`

Reply:
Hmm. I guess we lose order information that way.

Reply:
Let's roll with this as is. If we want to iterate later, we can.